jest.config.setup.files.after.env.js

```diff
@@ -0,0 +1,7 @@
+console.error = (message) => { // eslint-disable-line
+  // During test development, it might be useful to comment out
```

Comment:
It's probably a good idea to leave this note in one of our README or doc files, to make it more discoverable for new contributors. Or maybe a reference to this file?

Reply:
Done!